cmake/CreateLaunchers.cmake

```diff
@@ -235,11 +243,11 @@ macro(_launcher_produce_vcproj_user)
 
 		set(USERFILE_CONFIGSECTIONS)
 		foreach(USERFILE_CONFIGNAME ${config_types})
-            set(USERFILE_COMMAND "$<TARGET_FILE:${_targetname}>")
+      set(USERFILE_COMMAND "${COMMAND}")
```

Comment:
looks like an indentation error

Reply:
yeah, fix that and Ill merge it, do you use hunter? If so Ill version it and update hunter, if not likely wait.

Reply:
@caseymcc I don't use hunter.

The file has many indentation errors. To what should I standardize?